### PR TITLE
[PLAYER-5375] Update playhead on marker clicked at pause screen

### PR DESCRIPTION
**Pull request by marcosmx**
Added the needed code to update the playhead once the marker gets clicked.

### CODE REVIEW

**Comment by marcosmx** [comment]
@ooyala-amohnatkin test cases updated and passing now.

File: tests/components/markers/markerIcon-test.js [hunk 1]
```diff
@@ -273,7 +277,8 @@ describe('MarkerIcon component', function () {
     
     markerIcon.find('div.oo-marker-bubble.oo-marker-text').simulate('click');
     expect(seeked).toBe(true);
-    expect(seekTime).toBe(markerIcon.props().data.start);    
+    expect(seekTime).toBe(markerIcon.props().data.start);
+    expect(playheadUpdated).toBe(true);
```
Comment (KseniiaSokolovaEpam) [inline tests/components/markers/markerIcon-test.js]: There is a one more way to check if function was called. You could mock the function
```
updateSeekingPlayhead: jest.fn()
```
and after that use [toHaveBeenCalled](https://jestjs.io/docs/en/expect#tohavebeencalled) 
or [toHaveBeenCalledWith](https://jestjs.io/docs/en/expect#tohavebeencalledwith)
or .not.toHaveBeenCalled , etc

Comment (marcosmx) [reply tests/components/markers/markerIcon-test.js]: Thanks, I'll use it next time =)

Comment (ooyala-amohnatkin) [reply tests/components/markers/markerIcon-test.js]: @KseniiaSokolovaEpam let's leave it for now and refactor tests later

**Pull request merged**